bundles/org.openhab.binding.bambulab/src/main/java/org/openhab/binding/bambulab/internal/BambuLabBindingConstants.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bambulab.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link BambuLabBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Martin Grześlowski - Initial contribution
+ */
+@NonNullByDefault
+public class BambuLabBindingConstants {
+
+    public static final String BINDING_ID = "bambulab";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID PRINTER_THING_TYPE = new ThingTypeUID(BINDING_ID, "printer");
+
+    @SuppressWarnings("StaticMethodOnlyUsedInOneClass")
+    public interface Channel {
+        // PrintDetails
+        public static final String NOZZLE_TEMPERATURE_CHANNEL = "nozzleTemperature";
```

Comment:
I would prefer to have the channel part as prefix. Example: `CHANNEL_NOZZLE_TEMPERATURE` Could have been that i asked you to do the opposite with some of the other bindings you created. Anyway, it would make it easier to have a set of constants for channel, config, etc.